internal/compiler-bridge/src/main/scala_2.10/xsbt/Compat.scala

```diff
@@ -76,6 +79,20 @@ abstract class Compat {
     def enclosingTopLevelClass: Symbol = sym.toplevelClass
     def toplevelClass: Symbol = sourceCompatibilityOnly
     def asMethod: MethodSymbol = sym.asInstanceOf[MethodSymbol]
+
+    // Not present in 2.10
+    final def getterIn(base: Symbol): Symbol = sym.getter(base)
```

Comment:
Can we annotate these two also with `@inline`? Just in case, same comment as `unexpandedName`.